Console/Command/SpecificProductExport.php

```diff
@@ -0,0 +1,93 @@
+<?php
+declare(strict_types=1);
+
+namespace Aligent\FredhopperIndexer\Console\Command;
+
+use Aligent\FredhopperIndexer\Api\Export\ExporterInterface;
+use Aligent\FredhopperIndexer\Model\Export\Data\ImmediateProducts;
+use Magento\CloudDocker\Cli;
```

Comment:
Is this part of a standard M2 installation?

Reply:
Doubtful :thinking: